includes/embeds/class-amp-soundcloud-embed.php

```diff
@@ -116,7 +115,7 @@ private function parse_amp_component_from_iframe( $html, $url ) {
 	/**
 	 * Render shortcode.
 	 *
-	 * @todo Move this to Jetpack.
+	 * @deprecated 1.4.1 Moved to Jetpack in soundcloud_shortcode().
```

Comment:
With the deletion of `add_shortcode( ... )` above, this function is dead code. Maybe it'd be good to delete it entirely.

The [PR](https://github.com/Automattic/jetpack/pull/14028) to migrate Soundcloud shortcode to Jetpack doesn't add a function that this can detect, like:

```
if ( ! function_exists( 'jetpack_soundcloud_amp_shortcode' ) ) {
        add_shortcode( 'soundcloud', [ $this, 'shortcode' ] );
}